packages/components/molecules/f-user-message/src/components/UserMessage.vue

```diff
@@ -65,6 +71,11 @@ export default {
     color: $white;
     background-color: $orange;
     max-width: 100%;
+    margin: spacing(x2) 0;
```

Comment:
Inclined to agree with @dandel10n about outside margins being added outside the component if possible. Is it not possible to add these margins on the consuming side?

```
   <user-message class="c-home-messageBanner" />
```

And then add the styles to that class potentially?

Reply:
The way the styles are output by $style means I only get a generated class; I was originally going to do that as a "quick fix", the only way I could do this on the consuming side is to add elements to the DOM, and then add CSS to HomeWeb.

The way I see this, there are two valid cases to use this component in, flush and non flush; and it makes sense to me to make the component handle each with a prop so that its entirely self contained.

Reply:
Ill give that suggestion a go in Homeweb